tests/test_sync_pages.py

```
@@ -19,11 +19,11 @@ def test_parse_meta():
 
                 """
         )
-    ) == dict(
-        title="Jak na Git a GitHub",
-        description="Co je Git a k čemu se používá? Jaký je rozdíl mezi Gitem a GitHubem? Jak začít s Gitem?",
-        template="main_handbook.html",
-    )
+    ) == {
+        "title": "Jak na Git a GitHub",
```

Ruff (0.16.1)

[warning] String contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF001)